### ISSUE TITLE
Suggestion: Text Stays Open

### ISSUE THREAD

**Issue by y0himba** [Fosmos/Simply#2]
Is it possible to add a switch, or is there an edit I can do to keep the now playing text open even when not playing music?  It would just show the last thing that was played.

**Comment by Fosmos** [comment]
Yeah, I can add something, but for now if you're comfortable with a bit of coding. Open the animation.lua script in any text editor and navigate to the line 145 (the line above it says `-- Hide the song info`) and replace that block with this:
```
-- Hide the song info
--if table_contains(importantMeters, meter:GetName()) then
--update = InterpTowardHidden(i)
-- Show the cover and media controls
--else
update = InterpTowardHover(i)
 --end
```
So that it looks like this
![image](https://user-images.githubusercontent.com/59673626/155795979-57cc6cf5-19b0-47a3-bcbd-7480336d05fa.png)
